### ISSUE TITLE
LG C2 42 - turning OFF doesn't work after 03.30.16

### ISSUE THREAD

**Issue by hextorio** [JPersson77/LGTVCompanion#102]
I just noticed that after recent update the Companion is turning on my LG C2 just fine as before however turning off feature doesn't work anymore. Either when I press Test, Turn off or simply turn off my PC the display stays on.

Firmware update: 03.30.16

**Comment by JPersson77** [comment]
Hi @hextorio , 

I've not received that update yet myself so can't comment with my personal experiences. But from previous experience "turning it off and then on again" usually works to resolve. 

If this does not help your symptoms indicates that the pairing key for some reason was invalidated. Then you can try deleting and readding the device. Reinstalling won't help I think because that does not delete the config.

After that if it still does not work please paste the full log and config.json here so I can have a look.

**Comment by hextorio** [comment]
Here you go. Unfortunately deleting and adding the device again didn't work.

[Log.txt](https://github.com/JPersson77/LGTVCompanion/files/10437125/Log.txt)

{
    "Device1": {
        "Enabled": true,
        "HDMIinputcontrol": true,
        "IP": "192.168.1.106",
        "MAC": [
            "AC:5A:F0:73:0C:4D"
        ],
        "Name": "C2",
        "OnlyTurnOffIfCurrentHDMIInputNumberIs": 1,
        "SessionKey": "",
        "SetHDMIInputOnResume": true,
        "SetHDMIInputOnResumeToNumber": 1,
        "Subnet": "255.255.255.0",
        "WOL": 2
    },
    "LGTV Companion": {
        "AdhereDisplayTopology": false,
        "AutoUpdate": true,
        "BlankWhenIdle": true,
        "BlankWhenIdleDelay": 3,
        "ExtendedLog": true,
        "IdleFullscreen": false,
        "IdleWhiteListEnabled": false,
        "LocalEventLogRestartString": [
            "restart",
            "uruchomienie ponowne"
        ],
        "LocalEventLogShutdownString": [
            "shutdown",
            "power off",
            "wyłączenie zasilania"
        ],
        "PowerOffDuringRDP": false,
        "PowerOnTimeOut": 40,
        "Version": 2
    }
}

**Comment by JPersson77** [comment]
With a google translate of the error message it looks like the TV refuses the connection. Did you try pulling the powercord of the TV? Double check so the IP of the TV didn't change

**Comment by hextorio** [comment]
Pulling of the the powercord didn't work. IP is the same - it's fixed on the router to always be the same.

**Comment by JPersson77** [comment]
Try this and then try to pair again with the TV: 
General->Devices->External Devices->clear paired connections

**Comment by RoofDefenseSystems** [comment]
Yes, I recently received Software Version 03.30.14 I am currently installing the dev build to try and get you more info.

**Comment by RoofDefenseSystems** [comment]
[Log.txt](https://github.com/JPersson77/LGTVCompanion/files/10440787/Log.txt)

**Comment by weenan** [comment]
> I think it is possible LG changed something. Will investigate

Same issue here. 
Updated my C2 to 03.30.16 today and cannot for the life of me get the TV to shut down. It still powers on though.

**Comment by mbushnaq00** [comment]
I am facing the same issue as well, after updating my C2 to 03.30.16, the companion app can no longer pair with the TV (manually or automatically) even though I have a fixed IP in the DHCP server.
